canvas_modules/common-canvas/__tests__/common-properties/controls/toggle-test.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2017-2022 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import Toggle from "../../../src/common-properties/controls/toggle";
+import { mount } from "enzyme";
+import { expect } from "chai";
+import Controller from "../../../src/common-properties/properties-controller";
+import propertyUtils from "../../_utils_/property-utils";
+import toggleParamDef from "../../test_resources/paramDefs/toggle_paramDef.json";
+
+const controller = new Controller();
+
+const control = {
+	"name": "toggle",
+	"values": [
+		true,
+		false
+	]
+};
+
+propertyUtils.setControls(controller, [control]);
+
+const propertyId = { name: "toggle" };
+
+describe("toggle renders correctly", () => {
+
+	beforeEach(() => {
+		controller.setErrorMessages({});
+		controller.setControlStates({});
+		controller.setPropertyValues(
+			{ toggle: true }
+		);
+	});
+
+
+	it("toggle should set correct value", () => {
+		// In toggle.jsx we're setting toggled={this.props.value}
```

Comment:
This test doesn't really do anything.  You need to test where clicking on the toggle changes the value.